### PR TITLE
send email to core and unsub field

### PR DESCRIPTION
**Pull request by rolandpo**
- add additional email recipient through env var
- add `unsubscribed` field to subscriber model for analytics purposes
- error handling on invalid confirm/unsubscribe path params

issues #295, #297

**requires adding new env var CORE_TEAM_EMAIL set to core team email address**

### CODE REVIEW

**Review by sparrowDom — APPROVED** [review]
LGTM

File: eagleproject/core/blockchain/harvest/transaction_history.py [hunk 1]
```diff
@@ -1127,7 +1128,24 @@ def send_report_email(summary, report, prev_report, report_type):
             'email': subscriber.email,
             'conf_num': subscriber.conf_num
         }))
-        result = e.execute([subscriber.email])
+        e.execute([subscriber.email])
+
+
+def send_report_email_core(summary, report, prev_report, report_type):
+    core = settings.CORE_TEAM_EMAIL
+    e = Email(summary, "test", render_to_string('analytics_report_email.html', {
```
Comment (shahthepro) [inline eagleproject/core/blockchain/harvest/transaction_history.py:1136]: "test"??

**Review by shahthepro — APPROVED** [review]

File: eagleproject/core/blockchain/harvest/transaction_history.py (hunk 1 above)
Comment (sparrowDom) [reply eagleproject/core/blockchain/harvest/transaction_history.py:1136]: I never addressed this, this is plain_text version of the email ([see Django docs](https://docs.djangoproject.com/en/4.1/topics/email/#sending-alternative-content-types)) I would assume not many are still using email clients that are not able to render a html version of the email. 

We probably won't want to create double context for each email. Let me make a commit that strips HTML characters out of the rich content email and set that as plain text version

Comment (sparrowDom) [reply eagleproject/core/blockchain/harvest/transaction_history.py:1136]: done here: https://github.com/OriginProtocol/ousd-analytics/pull/298/commits/0797200bc9e0601218924220ea197905e4201a46

**Comment by sparrowDom** [comment]
CORE_TEAM_EMAIL added to prod and staging environments
merging and deploying

**Pull request merged**